TeachingRecordSystem/tests/TeachingRecordSystem.Api.Tests/V3/GetTrnRequestTests.cs

```diff
@@ -66,6 +67,82 @@ public async Task Get_TrnRequestNotFound_ReturnsNotFound()
         Assert.Equal(HttpStatusCode.NotFound, response.StatusCode);
     }
 
+    [Fact]
+    public async Task Get_MergedRecord_ReturnsExpectedResponse()
+    {
+        // Arrange
+        var requestId = Guid.NewGuid().ToString();
+        var firstName = Faker.Name.First();
+        var middleName = Faker.Name.Middle();
+        var lastName = Faker.Name.Last();
+        var dateOfBirth = new DateOnly(1990, 01, 01);
+        var email = Faker.Internet.Email();
+        var nationalInsuranceNumber = Faker.Identification.UkNationalInsuranceNumber();
+
+        var masterContact = await TestData.CreatePerson(p => p
+            .WithTrn(hasTrn: true)
+            .WithFirstName(firstName)
+            .WithMiddleName(middleName)
+            .WithLastName(lastName)
+            .WithDateOfBirth(dateOfBirth)
+            .WithEmail(email)
+            .WithNationalInsuranceNumber(nationalInsuranceNumber: nationalInsuranceNumber));
+
+        var existingContact = await TestData.CreatePerson(p => p
+            .WithTrn(hasTrn: false)
+            .WithFirstName(firstName)
+            .WithMiddleName(middleName)
+            .WithLastName(lastName)
+            .WithDateOfBirth(dateOfBirth)
+            .WithEmail(email)
+            .WithNationalInsuranceNumber(nationalInsuranceNumber: nationalInsuranceNumber));
+
+        await WithDbContext(async dbContext =>
+        {
+            dbContext.Add(new TrnRequest()
+            {
+                ClientId = ClientId,
+                RequestId = requestId,
+                TeacherId = existingContact.ContactId
+            });
+
+            await dbContext.SaveChangesAsync();
+        });
+
+
+        XrmFakedContext.UpdateEntity(new Contact()
```

Comment:
Probably better to send a `Microsoft.Crm.Sdk.Messages.MergeRequest` here for full-fidelity.